app/src/main/java/com/poupa/vinylmusicplayer/discog/DB.java

```diff
@@ -81,6 +83,7 @@ public void migrateDB(@NonNull SQLiteDatabase dbase, int oldVersion, int newVers
                 case 3:
                 case 4:
                 case 5:
+                case 6:
```

Comment:
To check: By resetting, do we lose the play history DB?

Reply:
Indeed, we lose the play history when a DB upgrade happens. I'll check if there is a way to preserve that info.

Reply:
Actually, I was wrong: this does not remove the play history / top tracks, etc. These are stored in the `HistoryStore`, which is another `.db` file.